runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkPipelineOptions.java

```diff
@@ -187,4 +188,12 @@
   Long getLatencyTrackingInterval();
 
   void setLatencyTrackingInterval(Long interval);
+
+  @Description(
+      "Flink mode for data exchange for batch pipeline. "
+          + "Reference {@link org.apache.flink.api.common.ExecutionMode}")
+  @Default.Enum("BATCH_FORCED")
+  ExecutionMode getExecutionModeForBatch();
```

Comment:
Can we add a test in `PipelineOptionsTest`?

Reply:
Sure

Reply:
TheDefault Enum test is done in ProxyInvocationHandlerTest.java
Please let me know if you are referring to some other test.

Reply:
I meant `PipelineOptionsTest` to test the default values. I'm adding it in the merge commit.

Reply:
Should this be getFlinkExecutionModeForBatch, as it seems rather Flink-specific?

Reply:
Considering the global namespace of the option names that would be preferred. However, not a single other option so far has a Flink prefix. Perhaps we have to come up with a way to scope option names?